packages/components/src/components/form/DateInput/types.ts

```diff
@@ -0,0 +1,13 @@
+import { MaskitoDateMode } from '@maskito/kit';
+import { InputProps } from '../Input';
+
+export interface DateInputProps extends InputProps {
+  maskOptions?: DateMaskOptions;
+}
+
+export type DateMaskOptions = {
+  mode?: MaskitoDateMode;
+  placeholder?: string;
+  separator?: string;
```

Comment:
can the separator be inferred from the `mode` property?, same as before, if we pass something different in this prop than the one used in `mode` the component will miss behave

Reply:
I don't think the `separator` can be inferred from the `mode` because we could have a `mm/dd/yyyy` mode working well with a `-` separator. In [here](https://maskito.dev/kit/date/API?mode=mm%2Fyyyy&separator=-) you can see the available `mode` and how it works with different separators